Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.yml

```diff
@@ -0,0 +1,1891 @@
+    deprecated: false
+    description: Push the candidate configuration.
+    execution: false
+    name: prisma-sase-candidate-config-push
+    polling: true
+    outputs:
+    - contextPath: PrismaSase.CandidateConfig.job_id
+      description: Configuration job ID.
+      type: String
+    - contextPath: PrismaSase.CandidateConfig.result
+      description: The configuration push result.
+      type: Boolean
+  - arguments:
+    - default: false
+      description: ID of the rule to be changed.
+      isArray: false
+      name: rule_id
+      required: true
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
+      isArray: false
+      name: folder
+      predefined:
+      - Shared
+      - Mobile Users
+      - Remote Networks
+      - Service Connections
+      - Mobile Users Container
+      - Mobile Users Explicit Proxy
+      required: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: pre
+      description: Security rule position.
+      isArray: false
+      name: position
+      predefined:
+      - pre
+      - post
+    - auto: PREDEFINED
+      default: false
+      description: Rule action.
+      isArray: false
+      name: action
+      predefined:
+      - deny
+      - allow
+      - drop
+      - reset-client
+      - reset-server
+      - reset-both
+    - default: false
+      description: The security rule's description.
+      isArray: false
+      name: description
+      required: false
+    - default: false
+      description: Rule log setting.
+      isArray: false
+      name: log_setting
+      required: false
+    - default: false
+      description: A comma-separated list of applications.
+      isArray: false
+      name: application
+      required: false
+    - default: false
+      description: A comma-separated list of categories.
+      isArray: true
+      name: category
+      required: false
+    - default: false
+      description: A comma-separated list of destination networks.
+      isArray: false
+      name: destination
+      required: false
+    - default: false
+      description: A comma-separated list of destination HIPs.
+      isArray: false
+      name: destination_hip
+      required: false
+    - default: false
+      description: Security profiles to apply to the traffic.
+      isArray: false
+      name: profile_setting
+      required: false
+    - default: false
+      description: Services the rule applies to.
+      isArray: false
+      name: service
+      required: false
+    - default: false
+      description: A comma-separated list of source networks.
+      isArray: true
+      name: source
+      required: false
+    - default: false
+      description: A comma-separated list of source HIPs.
+      isArray: false
+      name: source_hip
+      required: false
+    - default: false
+      description: A semi-colon (;) separated list of source user(s).
+      isArray: false
```

Comment:
If this is a comma-separated list, then should isArray be true?

Reply:
Not in this case. It is always a string. But changed all other ones